treeherder/webapp/api/resultset.py

```diff
@@ -133,6 +133,22 @@ def cancel_all(self, request, project, jm, pk=None):
         except Exception as ex:
             return Response("Exception: {0}".format(ex), 404)
 
+    @action(permission_classes=[IsAuthenticated])
+    @with_jobs
+    def fill_all(self, request, project, jm, pk=None):
+        """
+        Fill all the jobs in this resultset
+        """
+        if not pk:
+            return Response({"message": "resultset id required"}, status=400)
+
+        try:
+            jm.fill_all_revision_jobs(request.user.email, pk, project)
+            return Response({"message": "jobs triggered to be filled in for resultset '{0}' and branch '{1}'".format(pk, project)})
```

Comment:
> On treeherder we tend to avoid the term branch as much as possible because it can lead to confusion. What about
> Missing jobs triggered for result-set {resultset}

I'd like us to move back to using "push" instead of resultset if possible - seeing as the only reason we used resultset was for the multi-repo use-case which we're not using, and may possibly deprecate in the future?